http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterMonitorSupervisor.scala

```diff
@@ -58,7 +55,7 @@ object ClusterMonitorSupervisor {
   private case object CheckForClusters extends ClusterSupervisorMessage
 }
 
-class ClusterMonitorSupervisor(monitorConfig: MonitorConfig, dataprocConfig: DataprocConfig, clusterBucketConfig: ClusterBucketConfig, gdDAO: GoogleDataprocDAO, googleComputeDAO: GoogleComputeDAO, googleStorageDAO: GoogleStorageDAO, google2StorageDAO: GoogleStorageService[IO], dbRef: DbReference, authProvider: LeoAuthProvider, autoFreezeConfig: AutoFreezeConfig, jupyterProxyDAO: JupyterDAO, rstudioProxyDAO: RStudioDAO, welderProxyDAO: WelderDAO, leonardoService: LeonardoService, clusterHelper: ClusterHelper)
+class ClusterMonitorSupervisor(monitorConfig: MonitorConfig, dataprocConfig: DataprocConfig, clusterBucketConfig: ClusterBucketConfig, gdDAO: GoogleDataprocDAO, googleComputeDAO: GoogleComputeDAO, googleStorageDAO: GoogleStorageDAO, google2StorageDAO: GoogleStorageService[IO], dbRef: DbReference, authProvider: LeoAuthProvider, autoFreezeConfig: AutoFreezeConfig, jupyterProxyDAO: JupyterDAO, rstudioProxyDAO: RStudioDAO, welderProxyDAO: WelderDAO, clusterHelper: ClusterHelper)
   extends Actor with Timers with LazyLogging {
   import context.dispatcher
 
```

Comment:
Now that more of the cluster creation will be handled by the back leo, I wonder if we should try to use `TraceId` here as much as possible, too. Ideally, we could persist a `traceId` in the DB when the first record is created by the front Leo so monitors could retrieve and use it throughout the lifecycle of a cluster. I know it's work but it feels like it'd pay off when it comes to debugging. Alternatively, we could avoid the DB persistence work and create a new traceId for the back-Leo ops (perhaps by having `createChildActor` accept a `TraceId`). Even though we wouldn't be able to have a consistent `traceId` across the front- and back-Leos this way, it would be better than nothing.

Reply:
I have a different pattern for introducing `TraceId` in my other PR...so I'll leave it out here for now

Reply:
I agree, this is a good idea. When there are errors and back-leo persists to the `CLUSTER_ERROR` table, we could include a trace id to correlate with back-Leo logs. I'll try to make this happen -- thanks!

Reply:
Ohh thanks @Qi77Qi, I'll look at that.

Reply:
> I have a different pattern for introducing TraceId in my other PR...so I'll leave it out here for now

@Qi77Qi The pattern you mention is for how to pass around `TraceId`, correct? We would still need to persist a `TraceId` in DB to be able to use it across the lifecycle.